G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/ExcelSave/ShouldSaveExcelFile_WhenSaveCommandIsCalled.txt

```diff
@@ -0,0 +1,16 @@
+﻿♥filepath = ‴♥environment⟦TEMP⟧\tempexcelfile.xlsx‴
+file.exists filename ♥filepath errorjump SkipDelete timeout 2
```

Comment:
No need to check if the file exist. `file.delete` won't throw an exception if the file is not there.